homeassistant/components/halohome/const.py

```diff
@@ -0,0 +1,8 @@
+"""HALO Home integration constants."""
+from typing import Final
+
+DOMAIN: Final = "halohome"
+CONF_LOCATIONS: Final = "locations"
+PLATFORMS: Final = [
+    "light",
```

Comment:
Use The `Platform` enum here, e.g.,

```suggestion
    Platform.LIGHT,
```

<https://github.com/home-assistant/core/blob/dev/homeassistant/const.py#L22>